scipy/stats/mstats_basic.py

```diff
@@ -2300,7 +2300,9 @@ def skew(a, axis=0, bias=True):
             m3 = np.extract(can_correct, m3)
             nval = ma.sqrt((n-1.0)*n)/(n-2.0)*m3/m2**1.5
             np.place(vals, can_correct, nval)
-    return vals
+    # Add 0 to ensure a scalar result is returned
```

Comment:
should the docstring at L2280 should be updated?

```
    Returns
    -------
    skewness : ndarray
        The skewness of values along an axis, returning 0 where all values are
        equal.
```

it seems in this case it's returning a scalar instead of a ndarray

Reply:
```
In [30]: skew([[1,5,4],[5,1,3],[4,7,nan]], nan_policy="omit") + 0   
Out[30]: 
masked_array(data=[-0.52800498, -0.38180177,  0.        ],
             mask=False,
       fill_value=1e+20)
```
Will the new code handle this case? maybe `vals.data` is what we actually want?

Reply:
I'm not sure what we want in that case. To me the problem wasn't from the masked array - it is that an array is being returned at all when there is only a single dimension.

Reply:
`vals.data` seems to correctly be a scalar vs ndarray. spitballing it looks like

```
vals = vals.data
if fisher:
    return vals - 3
else:
    return vals
```
Seems to work

Reply:
I've just run the above and it still gives a float value for fisher and an array (not a masked array) for the regular version.

```python
>>> import numpy as np
>>> from scipy.stats import kurtosis
>>> kurtosis([1, 5, 4, 6, np.nan], nan_policy="omit")
-1.0
>>> kurtosis([1, 5, 4, 6, np.nan], fisher=False, nan_policy="omit")
array(2.)
```
Would we expect both results to be a float instead?